js/server/modules/@arangodb/foxx/router/swagger-context.js

```diff
@@ -470,14 +470,8 @@ module.exports = exports =
         operation.produces.push('application/json');
       }
 
-      for (const param of this._pathParams.entries()) {
-        const name = param[0];
-        const def = param[1];
-        const parameter = (
-        def.schema
-          ? swaggerifyParam(def.schema.isJoi ? def.schema : joi.object(def.schema))
-          : {type: 'string'}
```

Comment:
There's now always some kind of schema (defaulting to the string schema) so the ternary is no longer necessary here.